src/miner_lora.erl

```diff
@@ -166,33 +166,47 @@ reg_domain_data_for_addr(Addr, #state{chain=Chain}) ->
         undefined ->
             {error, no_ledger};
         Ledger ->
-            %% check if the poc 11 vars are active yet
-            case blockchain_ledger_v1:find_gateway_location(Addr, Ledger) of
-                {ok, Location} ->
-                    case blockchain_region_v1:h3_to_region(Location, Ledger) of
-                        {ok, Region} ->
-                            case blockchain_region_params_v1:for_region(Region, Ledger) of
-                                {ok, RegionParams} ->
-                                    {ok, {Region, [ (blockchain_region_param_v1:channel_frequency(RP) / ?MHzToHzMultiplier) || RP <- RegionParams ]}};
-                                {error, Reason} ->
-                                    {error, Reason}
-                            end;
-                        {error, regulatory_regions_not_set} ->
-                            case country_code_for_addr(Addr) of
-                                {ok, CC} ->
-                                    %% use country code to get regulatory domain data
-                                    reg_domain_data_for_countrycode(CC);
+            case blockchain:config(poc_version, Ledger) of
+                {ok, V} when V > 10 ->
+                    %% check if the poc 11 vars are active yet
+                    case blockchain_ledger_v1:find_gateway_location(Addr, Ledger) of
```

Comment:
this would be a great place for EEP 49 if it's ever officially adopted

Reply:
yea, it's not good. Check some of the case ladders in core for maximum annoyance